docs/docs/api.md

```diff
@@ -347,23 +374,76 @@ If you have questions, come chat with us on [Discord](https://discord.com/invite
   }
   ```
 
-### `/v0/slug/[marketSlug]`
+### `GET /v0/slug/[marketSlug]`
+
+Gets information about a single market by slug (the portion of the URL path after the username).
 
-This is a convenience endpoint for getting info about a market from it slug (everything after the last slash in a market’s URL).
+Requires no authorization.
 
 - Example request
   ```
   https://manifold.markets/api/v0/slug/will-carrick-flynn-win-the-general
   ```
 - Response type: A `FullMarket` ; same as above.
 
-## Deprecated
+### `POST /v0/bet`
+
+Places a new bet on behalf of the authorized user.
+
+Parameters:
+
+- `amount`: Required. The amount to bet, in M$, before fees.
+- `contractId`: Required. The ID of the contract (market) to bet on.
+- `outcome`: Required. The outcome to bet on. For binary markets, this is `YES`
+  or `NO`. For free response markets, this is the ID of the free response
+  answer. For numeric markets, this is a string representing the target bucket,
```

Comment:
Nice work for even including numeric markets! I think we will probably change them soon to be simple and no longer take a `value` param, but for now this is good.